src/api/queries/getMetricQueries.js

```diff
@@ -1,33 +1,35 @@
 var TIME_BOUNDS = ['1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
-var CURRENCIES = ['usd']
+var SUPPORTED_CURRENCIES = ['USD']
+var DEFAULT_CURRENCY = 'USD'
 
 function currencyOptionChecker_ (options) {
-  if ('currency' in options) {
-    options.currency = options.currency || 'usd'
-  }
+  if ('currency' in options) { options.currency = options.currency || DEFAULT_CURRENCY }
 }
 
 function timeboundOptionChecker_ (options) {
-  if ('timeBound' in options) {
-    options.timeBound = options.timeBound || ''
-  }
+  if ('timeBound' in options) { options.timeBound = options.timeBound || '' }
 }
 
 function optionsChecker_ (options) {
   currencyOptionChecker_(options)
   timeboundOptionChecker_(options)
```

Comment:
I'm not a fan of such use of pass by reference. Maybe we could indicate it by the function name `currencyOptionModiy_` ?

Reply:
If I read `checker` I expect something that just returns true/false. Best option would be imo `options.currency = currencyOptionModify_(options)`